### PR TITLE
Update TOC to remove  CLI preview

### PR DESCRIPTION
**Pull request by CarolKigoonya**
Graph CLI is now GA

### CODE REVIEW

File: concepts/toc.yml [hunk 1]
```diff
@@ -1190,4 +1190,4 @@ items:
         displayName: Microsoft Graph REST API v1.0 reference
       - name: API beta reference
         href: /graph/api/overview?view=graph-rest-beta&preserve-view=true
```
Comment (SteveMutungi254) [inline concepts/toc.yml:1192]: Kindly remove the text after the "question mark". Use this link instead `/graph/api/overview`. Acrolinx should have thrown a warning too.

**Review by SteveMutungi254 — CHANGES_REQUESTED** [review]
Left a minor suggestion.

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Comment by csmulligan** [comment]
Hi @CarolKigoonya I found the same file in _api-reference\v1.0\toc.yml_ and _api-reference\beta\toc.yml_ with the preview label. Could you please remove the label from those files too? 
Thanks,
Csilla

**Comment by SteveMutungi254** [comment]
> Hi @CarolKigoonya I found the same file in _api-reference\v1.0\toc.yml_ and _api-reference\beta\toc.yml_ with the preview label. Could you please remove the label from those files too? Thanks, Csilla

Good catch, @csmulligan

**Review by Lauragra — APPROVED** [review]

**Pull request merged**